exercises/03-sagas/after/Divergent.Shipping/Sagas/ShippingSaga.cs

```diff
@@ -7,16 +7,20 @@
 
 namespace Divergent.Shipping.Sagas
 {
-    class ShippingSaga : Saga<ShippingSagaData>,
+    using NServiceBus.Persistence.Sql;
+
+    class ShippingSaga : SqlSaga<ShippingSagaData>,
```

Comment:
This is the change that we wanted to avoid. Does the new version of SQLP not support standard core sagas?